docker/README.md

```diff
@@ -200,14 +200,18 @@ Delete Volume
 docker volume rm zwavejs2mqtt
 ```
 
-## Custom builds
+# Custom builds
 
-The docker images are the latest stable images of the [zwavejs2mqtt](https://github.com/zwave-js/zwavejs2mqtt) repo. If you want to keep your image updated with the latest changes you can build it on your local machine. Just select a commit sha, a branch name, or a tag name, and pass it to docker build using the _--build-arg_ option for the _Z2M_GIT_SHA1_ argument. For example:
+There are two methods to build a custom docker container, using either the project's Dockerfile or Dockerfile.contrib. While the Dockerfile method offers a faster build process, the alternative Dockerfile.contrib method allows you to specify the specific branches for node-zwave-js and zwavejs2mqtt, and even the ability to specify alternative repositories (such as your own).
+## Building a container using Dockerfile
+
+The docker images are the latest stable images of the [zwavejs2mqtt](https://github.com/zwave-js/zwavejs2mqtt) repo. If you want to keep your image updated with the latest changes you can build it on your local machine. For example:
 
 ```bash
 git clone https://github.com/zwave-js/zwavejs2mqtt.git
 cd zwavejs2mqtt
-docker build -f docker/Dockerfile --build-arg Z2M_GIT_SHA1=master -t zwavejs/zwavejs2mqtt:latest .
```

Comment:
@robertsLando The documentation says that you can provide Z2M_GIT_SHA1 as a build arg, but that arg isn't actually implemented in the Dockerfile anywhere. Unless I'm missing something, I'm not sure how it could be used as those only appear to be effective to override an ARG in the Dockerfile.

Reply:
probably an oversight. Using a git SHA as the `Z2M_BRANCH` or `ZJS_BRANCH` should work just like a branch name.

Reply:
> probably an oversight. Using a git SHA as the Z2M_BRANCH or ZJS_BRANCH should work just like a branch name.

Yeah that's how it is supposed to work.

Reply:
I think that is left from some prev versions of that files